<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>580</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QWidget" name="MainWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="VideoScreenLabel">
         <property name="frameShape">
          <enum>QFrame::WinPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="FunctionWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="ProcessWidget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="leftMargin">
              <number>5</number>
             </property>
             <property name="rightMargin">
              <number>5</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="CurrentTimeLabel">
               <property name="text">
                <string>0:00</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="ProgressSlider">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="TotalTimeLabel">
               <property name="text">
                <string>0:00</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="ButtonWidget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="leftMargin">
              <number>5</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>5</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QWidget" name="ModeWidget" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QToolButton" name="ModeButton">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
                  </property>
                  <property name="text">
                   <string>单曲播放</string>
                  </property>
                  <property name="icon">
                   <iconset resource="images.qrc">
                    <normaloff>:/images/PlaySingle.png</normaloff>:/images/PlaySingle.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="VoiceButton">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
                  </property>
                  <property name="text">
                   <string>音量</string>
                  </property>
                  <property name="icon">
                   <iconset resource="images.qrc">
                    <normaloff>:/images/Voice.png</normaloff>:/images/Voice.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="SpeadButton">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="images.qrc">
                    <normaloff>:/images/Set.png</normaloff>:/images/Set.png</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QWidget" name="PlayWidget" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QToolButton" name="LastVedioButton">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
                  </property>
                  <property name="text">
                   <string>上</string>
                  </property>
                  <property name="icon">
                   <iconset resource="images.qrc">
                    <normaloff>:/images/Next.png</normaloff>:/images/Next.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="PlayButton">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="images.qrc">
                    <normaloff>:/images/Play.png</normaloff>:/images/Play.png</iconset>
                  </property>
                  <property name="checkable">
                   <bool>false</bool>
                  </property>
                  <property name="popupMode">
                   <enum>QToolButton::DelayedPopup</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="NextVedioButton">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
                  </property>
                  <property name="text">
                   <string>下</string>
                  </property>
                  <property name="icon">
                   <iconset resource="images.qrc">
                    <normaloff>:/images/Last.png</normaloff>:/images/Last.png</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QWidget" name="ControlWidget" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QToolButton" name="FullScreenButton">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
                  </property>
                  <property name="text">
                   <string>全屏</string>
                  </property>
                  <property name="icon">
                   <iconset resource="images.qrc">
                    <normaloff>:/images/FullScreen.png</normaloff>:/images/FullScreen.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="toolButton">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="images.qrc">
                    <normaloff>:/images/Plus.png</normaloff>:/images/Plus.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="PlayListButton">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
                  </property>
                  <property name="text">
                   <string>播放列表</string>
                  </property>
                  <property name="icon">
                   <iconset resource="images.qrc">
                    <normaloff>:/images/PlayList.png</normaloff>:/images/PlayList.png</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="PlayListWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QScrollArea" name="ScrollArea">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::WinPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="ScrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>278</width>
            <height>532</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QListWidget" name="ListWidget">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="focusPolicy">
              <enum>Qt::StrongFocus</enum>
             </property>
             <property name="frameShape">
              <enum>QFrame::HLine</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
